src/HotChocolate/Core/test/Types.Scalars.Tests/EmailAddressTypeTests.cs

```diff
@@ -0,0 +1,145 @@
+using System;
+using HotChocolate.Language;
+using Snapshooter.Xunit;
+using Xunit;
+
+namespace HotChocolate.Types.Scalars
+{
+    public class EmailAddressTypeTests : ScalarTypeTestBase
+    {
+        [Fact]
+        public void Schema_WithScalar_IsMatch()
+        {
+            // arrange
+            ISchema schema = BuildSchema<EmailAddressType>();
+
+            // act
+            // assert
+            schema.ToString().MatchSnapshot();
```

Comment:
Match snapshot creates a file in __snapshots__  on first run and compares the value against this file on later runs

If the snapshot comparison fails, a file is created in __snapshots__/__mismatch__ 

this snapshot file is missing :) does this not wokr on your machine?